postgresql/files/9.5/postgresql.conf.Debian

```diff
@@ -225,6 +225,23 @@ dynamic_shared_memory_type = posix	# the default is the first option
 
 # Set these on the master and on any standby that will send replication data.
 
+{%- if cluster.enabled and cluster.role != "master" %}
+hot_standby = on
+{%- endif %}
+
+{%- if cluster.enabled and cluster.role == "master" %}
+
+wal_level = hot_standby
```

Comment:
wal_level has exactly one working setting for both cluster solutions so I removed it from the metadata instead and kept it hardcoded per cluster solution